index.html

```diff
@@ -12,64 +12,73 @@
 
 <body>
     <section class="container">
-        <header>
-           <div class="logo"> <a>Dorcas</a></div>
-            <nav class="Union">
-                <ul>
-                    <li><a href="#portofolio" target="_blank">Portfolio</a></li>
-                    <li><a href="#about" target="_blank">About</a></li>
-                    <li><a href="#contact" target="_blank">Contact</a></li>
-                </ul>
-                <i class="fa fa-list-squares"></i>
-            </nav>
```

Comment:
- [ ] You did well here 😃 at adding the icon, but please decrease the distance between the icon and the links as you see here the margin between them is greater than the Figma Design 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/224803177-f969ead0-d573-4945-857c-db3afbad6894.png)    |   ![image](https://user-images.githubusercontent.com/83360636/224803265-1baf5f7a-e819-441b-90ff-8fbd3a09cd31.png) |